ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -148,14 +148,6 @@ tests:
         NETWORK_TYPE=OVNKubernetes
       EXTRA_MG_ARGS: --host-network
     workflow: baremetalds-e2e-ovn-ipsec
-- as: e2e-aws-ovn-windows
```

Comment:
consider marking the job as optional (not required), instead.

cc @openshift/openshift-team-windows-containers